### DISCUSSION TITLE
Need help with Patched talkgroups

### DISCUSSION THREAD

**Discussion by infusopharm** [chuot/rdio-scanner#294] [Q&A]
I have figured out how to use a range of talkgroups and make them record in SDR Trunk.  The files are generating great.  During slow times, dispatch on the P25 system patches at least two channels together into a new talkgroup in the range of 65100-65199.  I can have it auto populate everytime a new talk group is used but is there are better way to manage patched groups.  Would rather not have 100 different talkgroup displayed when only one is in use at a time.
Thanks in advance

**Comment by chuot** [comment]
I'll see if I can make something about that with the upcoming v7.
